scripts/retrieve_prs.py

```diff
@@ -20,21 +18,16 @@
 )
 
 
-def run(command):
-    """Returns (return-code, stdout, stderr)"""
-    p = subprocess.Popen(command, stdout=subprocess.PIPE, stderr=subprocess.PIPE, shell=True)
-    output, err = p.communicate()
-    rc = p.returncode
-    enc = locale.getpreferredencoding()
-    output = output.decode(enc)
-    err = err.decode(enc)
-    return rc, output.strip(), err.strip()
+def _run_cmd(cmd):
+    try:
+        return subprocess.check_output(cmd).strip()
+    except Exception:
+        return None
 
```

Comment:
No `try ~ except` here. The purpose of the script is to purse, so if it cannot, it should stop.

Reply:
Also I think you need `decode("utf-8")`

Reply:
Okay, let me address it in the previous PR.